nodes/3d_armor_stand.lua

```
@@ -5,18 +5,60 @@ local S = wrench.translator
 
 local def = minetest.registered_nodes["3d_armor_stand:armor_stand"]
 
-local add_entity_and_node = def and def.after_place_node
-local update_entity = def and def.on_metadata_inventory_put
+local function noop() end
```

Both functions still exist upstream, why is this fallback needed?

---

'needed' per se, no. It felt wrong when I read the code, that a possible nil value would be attempted to be called as function later on. I'm OK with leaving that part as it was if you think its better to fail loudly in the case that the methods were removed.

I figured since this mod supports so many mods, having it *not* crash because one changed would be preferable.

---

I'm OK with reverting that part though, if you think it is better to crash than to at least work for all the other supported mods.

---

I'd personally prefer not to fail silently to avoid 'hidden' bugs.
You decide :-)

---

could also have noop() emit warning to logs. Removed it.